index.html

```diff
@@ -220,7 +220,23 @@ <h4 id="popup-title"></h4> <i class="far fa-window-close" id="closePopup"></i>
               <ul class="skills" id="popup-skills">
               </ul>
```

Comment:
- [ ] Please in the about section the left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right in your code the text much more space than than the framework and tech section

|Yours  | Figmas |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/215270332-dbd08b81-ed7e-49cf-ab75-99c6b088e3ba.png)  | ![image](https://user-images.githubusercontent.com/74744787/215270320-2ec068bd-f90b-4a50-857a-4897c3813344.png)  |